docs/bold-bi-auto-deployment.md

```diff
@@ -0,0 +1,982 @@
+              BOLD_SERVICES_BRANDING_FOOTER_LOGO: "https://i.postimg.cc/FRyqDKPT/branding-header-logo.png "
+              
+              # This is organization name.
+              BOLD_SERVICES_SITE_NAME: "Autodeployment"
+              
+              # This is site identifier, and it will be the part of the application URL.
+              BOLD_SERVICES_SITE_IDENTIFIER: "branding"
+
+              # Note: If you want to use the custom branding, provide the value for all branding variables 
+              # If all variable values are given, application will use the branding images,
+              # otherwise it will take the default logos.
+            ---     
+        </details>
+
+           
+    * If you need to use **Bing Map** widget feature, enter value for `widget_bing_map_enable` environment variable as `true` and API key value for `widget_bing_map_api_key`.
+       
+       ![Enable-Bingmap](images/deploy/aks/bingmap_enable.png)
+
+6. If you have a DNS to map with the application, then you can continue with the following steps, else skip to **Step 11**. 
+
+7. Uncomment the host value and replace your DNS hostname with `example.com` in deploy_aks.yaml file in line **1442**.
+
+      ![DNS](images/deploy/aks/ingress_http_yaml.png)
+
+8. If you have the SSL certificate for your DNS and need to configure the site with your SSL certificate, follow the below step or you can skip to **Step 11**.
+
+9.  Run the following command to create a TLS secret with your SSL certificate.
+
+```sh
+kubectl create secret tls bold-tls -n bold-services --key <key-path> --cert <certificate-path>
+```
+
+10. Now, uncomment the `tls` section and replace your DNS hostname with `example.com` in ingress spec and save the file.
+
+       ![ingress DNS](images/deploy/aks/ingress_yaml.png)
+
+11. Now, run the following command to get the ingress IP address.
+
+      ```sh
+      kubectl get svc -n ingress-nginx
+      ```
+      Repeat the above command till you get the IP address in EXTERNAL-IP tab as shown in the following image. 
+      ![Ingress Address](images/deploy/aks/ingress_address.png) 
+
+12. Note the EXTERNAL-IP address and map it with your DNS, if you have added the DNS in **deploy_aks.yaml** file. If you do not have the DNS and want to use the application, then you can use the EXTERNAL-IP address.
+
+13. Replace your DNS or EXTERNAL-IP address in `<application_base_url>` place.
+
+    Ex:  `http://example.com`, `https://example.com`, `http://<external_ip_address>`
+    
+    ![App_Base_Url](images/deploy/aks/app_base_url.png) 
+    
+14. Read the optional client library license agreement from the following link.
+
+    [Consent to deploy client libraries](../docs/consent-to-deploy-client-libraries.md)
+    
+15. By default all the client libraries will be installed for Bold BI in Kubernetes. Still you can still overwrite them by mentioning the required libraries as comma separated like below in the environment variable noted from the above link.
```

Comment:
By default, all client libraries for Bold BI will be installed in Kubernetes. However, you can overwrite them by specifying the required libraries as a comma-separated list in the environment variable noted from the above link.

Reply:
concern addressed